tiberius/src/protocol/codec/token/token_row.rs

```diff
@@ -6,13 +6,38 @@ use crate::{
     },
 };
 use std::sync::Arc;
+use tokio::io::AsyncReadExt;
 
 #[derive(Debug)]
 pub struct TokenRow {
     pub meta: Arc<TokenColMetaData>,
     pub columns: Vec<ColumnData<'static>>,
 }
 
+struct RowBitmap {
+    data: Vec<u8>,
+}
+
+impl RowBitmap {
+    fn is_null(&self, i: usize) -> bool {
+        let index = i / 8;
+        let bit = i % 8;
+
+        self.data[index] & (1 << bit) > 0
+    }
+
+    async fn decode<R>(src: &mut R, columns: usize) -> crate::Result<Self>
+    where
+        R: AsyncReadLeExt + Unpin,
+    {
+        let size = (columns + 8 - 1) / 8;
+        let mut data = vec![0; size];
+        src.read_exact(&mut data[0..size]).await?;
+
+        Ok(Self { data })
```

Comment:
I think it works differently on postgres (idk in the protocol but at least the storage), only the nullable columns are taken into account for the size of the bitmap.